### PR TITLE
getSubmissionData

### PR DESCRIPTION
**Pull request by jeffreykat**
Added the rest of Submission data. More changes needed for sample after review.

### CODE REVIEW

File: app/src/main/java/com/myvetpath/myvetpath/CreateSubActivity.java [hunk 1]
```diff
@@ -288,28 +279,27 @@ private boolean isInteger(String str) {
...
+    //Todo: Add checks for empty inputs
+    private boolean loadSubmissionData(int status, Submission newSub){
+        long curDate = Calendar.getInstance().getTime().getTime();
         int numberOfSamples;
-        String submissionTitle = ((EditText) findViewById(R.id.sub_title)).getText().toString();
-        String groupName = ((EditText) findViewById(R.id.group_name_ET)).getText().toString();
         String sampleLocation = ((EditText) findViewById(R.id.location_sample_ET)).getText().toString();
 
         //check if user entered an integer into the number of samples text field. We many want to add some input validation later
+        //TODO: Change this to a NumberPicker to avoid this problem
```
Comment (Bradford-Wong) [inline app/src/main/java/com/myvetpath/myvetpath/CreateSubActivity.java]: I wasn't aware that this was a thing, great idea!

**Comment by Bradford-Wong** [comment]
Oh and don't worry about anything sample related. I'll take care of that.

**Comment by jeffreykat** [comment]
I ended up using random numbers for the FK sick element IDs, we should figure out a better way.

**Comment by Bradford-Wong** [comment]
Thanks for getting this done! Yeah we could set the sick element's internalID to be whatever the submission's internalID is and delete the sickElementID in the submission. If you're interested, I did do this for the samples and pictures. Personally, I'm OK with making a note about this and changing it later, but you can change it now if you want. Great work!

**Pull request merged**